procedures/unit-testing-basics.ipf

```diff
@@ -1238,8 +1245,420 @@ static Function ExecuteHooks(hookType, hooks, juProps, name, procWin, [param])
 	endswitch
 End
 
+/// @brief Background monitor of the Unit Testing Framework
+Function $"UTFBackgroundMonitor"(s)
+	STRUCT WMBackgroundStruct &s
+
+	variable i, numTasks, result, stopState
+	string task
+
+	DFREF df = GetPackageFolder()
+	SVAR/Z tList = df:BCKG_TaskList
+	SVAR/Z rFunc = df:BCKG_ReentryFunc
+	NVAR/Z timeout = df:BCKG_EndTime
+	NVAR/Z mode = df:BCKG_Mode
+
+	if(!SVAR_Exists(tList) || !SVAR_Exists(rFunc) || !NVAR_Exists(mode) || !NVAR_Exists(timeout))
+		print "Fatal: UTFBackgroundMonitor can not find monitoring data in package DF, aborting monitoring."
+		return 2
+	endif
+
+	result = !(mode & BACKGROUNDMONMODE_OR)
+	numTasks = ItemsInList(tList)
+	for(i = 0; i < numTasks; i += 1)
+		task = StringFromList(i, tList)
+		CtrlNamedBackground $task, status
+		stopState = !NumberByKey("RUN", S_Info)
+		if(mode & BACKGROUNDMONMODE_OR)
+			result = result | stopState
+		else
+			result = result & stopState
+		endif
+	endfor
+	result = (timeout && datetime > timeout) ? 1 : result
+
+	if(result)
+		RunTest("")
```

Comment:
@MichaelHuth How about `RunTest("UNUSED_FOR_REENTRY")` just to not confuse a future reader?

Reply:
Ok I can add a StrConstant, though it should have a content that is non valid as function name, like
:UNUSED_FOR_REENTRY: